CONTRIBUTING.md

```diff
@@ -62,43 +62,43 @@ For your convenience all the packages are linked. So, for example, if you run `e
 On the other hand, **badly written tests** that depend on the implementation will need to be changed any time the implementation changes, **causing frustration and unnecessary work**, while barely adding any value.
 
 In order to achieve good quality tests, **please follow the following principles**:
 - Always test the **behavior** of a component, not the implementation.
-    - Tests that use *Jest snapshots* almost always violate this. (Except maybe when you are testing an AST parser, a linter or similar.) 
+    - Tests that use *Jest snapshots* almost always violate this. (Except maybe when you are testing an AST parser, a linter, or similar.) 
     
     *Do instead:* Extract the real business requirement from what would be the snapshot and assert against that.
     
     - Tests that `find` child components and assert against props being passed are mostly wrong. 
     Use the **recommended selectors** (see point below) and **[`jest-dom` assertions](https://github.com/testing-library/jest-dom)** to enforce constraints that actually matter to the user.
-    - Searching for DOM elements using tag name, CSS class or hierarchy (`parentElement`, etc.) is an antipattern.
+    - Searching for DOM elements using tag name, CSS class, or hierarchy (`parentElement`, etc.) is an antipattern.
 
     *Do instead:* use **`findByRole` or other queries from [TL's query hierarchy](https://testing-library.com/docs/queries/about#priority)**. 
-    Feel free to add accessibility attributes where missing. With a bit of practice you'll see that almost everything can be covered with `*byRole`. Warning: it's addictive. ;)
+    Feel free to add accessibility attributes where missing. With a bit of practice, you'll see that almost everything can be covered with `*byRole`.
 - The goal for your test suite is **to cover** as much of the **business requirements** (e.g. in the issue description) as practical.
 - An ideal test suite only requires a change **if business requirements change**.
 - While not generally expected, **it is OK to add additional tests** for helper functions, hooks and subcomponents where it would not be practical to test them alongside their host components.
 In this case, still try to follow the principles above as much as practical.
-- Use **Jest/JSDOM testing wherever suitable**. Cypress-based tests should be reserved for high level integration tests, such as:
+- Use **Jest/JSDOM testing wherever suitable**. Cypress-based tests should be reserved for high-level integration tests, such as:
     - Basic sanity checks to make sure that *Elements* builds and loads correctly in a certain environment.
-    - Making sure that the routers/navigation of the external framework play nicely with our internal router.
+    - Making sure that the routers/navigation of the external framework plays nicely with our internal router.
```

Comment:
I would just make this "make sure the routers of the external framework play nicely with our internal router."